qcodes/tests/drivers/test_lakeshore_325.py

```diff
@@ -0,0 +1,27 @@
+import numpy as np
+from hypothesis import given
+import hypothesis.strategies as st
+from qcodes.instrument_drivers.Lakeshore import Model_325
+
+COMPONENTS = [0, 1, 16, 32, 64, 128]
```

Comment:
This is much better. Thanks!

The `list` doesn't add anything, can I just use the following?
`COMPONENTS = sorted(Model_325.Model_325_Sensor.sensor_status_codes.keys())`

Reply:
sure, remove the `list` :)

Reply:
I removed the dictionary `sensor_status_codes` from the sensor class because it is not used in the decoder anymore, as a result, I added the dictionary in the test function. (no way to call the IntFlag object in the decoder)